docs/how-to-guides/connecting-from-go.md

```diff
@@ -223,11 +223,21 @@ message type. We will use the wrapped ugnot realm for this example, wrapping
 `1000000ugnot` (1 $GNOT) for demonstration purposes.
 
 ```go
```

Comment:
What is this random import for?

Reply:
I'm following the example from the doc. The section above adds new code that uses `crypto.AddressFromBech32`, so the section also instructs the user to [add more imports](https://github.com/gnolang/gno/blob/d1c2b3fbc5e50a38eeffdb31b650cb9e8e9d9234/docs/how-to-guides/connecting-from-go.md?plain=1#L169-L171).

Reply:
(So, the code I add in this section needs `ugnot`, `vm` and `std`, so I need to show the imports for these.

Reply:
I don't think we ultimately need this, but let's leave it for now